release/models/platform/openconfig-platform-types.yang

```diff
@@ -360,4 +383,56 @@ module openconfig-platform-types {
       "A generic type reflecting whether a hardware component
       is powered on or off";
   }
+
+  identity COMPONENT_SWITCHOVER_REASON {
+    description
+    "For components that have redundant roles (e.g. two
+    supervisors in a device, one as master the other as backup),
+    this is the base entity for the role switchover reason of
+    the component.";
+  }
+
+  identity SWITCHOVER_USER_INITIATED {
+    base COMPONENT_SWITCHOVER_REASON;
+    description
+      "User initiated the switchover of the componenent's
+      master/backup role.";
+  }
+
+  identity SWITCHOVER_LOST_MASTER {
+    base COMPONENT_SWITCHOVER_REASON;
+    description
+      "The component assumes master role due
+      to lost of communication with component of the master role.";
```

Comment:
Would this identity be used if the user powered the master node off (rather than invoking an explicit switchover action)?  If so, perhaps worth clarifying in the description?

Reply:
Thanks. My response and code update to your other comment has addressed this comment. :) Marking it as resolved. Thanks!